Week2/homework/kalb3/maartjes-work.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/* cSpell:disable */
+
+const monday = [
+  {
+    name: 'Write a summary HTML/CSS',
+    duration: 180,
+  },
+  {
+    name: 'Some web development',
+    duration: 120,
+  },
+  {
+    name: 'Fix homework for class10',
+    duration: 20,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+];
+
+const tuesday = [
+  {
+    name: 'Keep writing summary',
+    duration: 240,
+  },
+  {
+    name: 'Some more web development',
+    duration: 180,
+  },
+  {
+    name: 'Staring out the window',
+    duration: 10,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+  {
+    name: 'Look at application assignments new students',
+    duration: 40,
+  },
+];
+
+const maartjesTasks = monday.concat(tuesday);
+const maartjesHourlyRate = 20;
+
+function computeEarnings(tasks, hourlyRate) {
+  tasks = tasks
+    .map(durations => durations.duration / 60)
+    .filter(durations => durations >= 2)
+    .map(durations => durations * hourlyRate)
+    .reduce((x, y) => x + y);
+  return tasks;
+}
+
+const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
+let earn = Math.floor(earnings * 100) / 100.0;
+earn = new Intl.NumberFormat('nl-NL', { style: 'currency', currency: 'EUR' }).format(earnings);
```

Comment:
Cool use of the `Intl.NumberFormat` function! You are not using the output from the line above (`let earn = Math.floor(earnings * 100) / 100.0;`), since you are using the `earnings` value here. You could skip that line and make this a `const`.